spice/ovirt/lib4x/user_portal/basictab.py

```diff
@@ -638,9 +645,10 @@ def wait_until_vm_starts_booting(self, name, timeout=None):
         return self._wait_for_vm_status(name, 'is_booting', timeout)
 
     def get_vms_names(self):
-        marker = '//div[starts-with(@id, "MainTabBasicListView_vm")][contains(@id, "_name")]'
+        marker = ('//div[starts-with(@id, '
+                  '"MainTabBasicListView_vm")][contains(@id, "_name")]')
         vms = self.driver.find_elements(by.By.XPATH, marker)
-        vms_names = map(lambda x: getattr(x, 'text'), vms)
+        vms_names = [getattr(x, 'text') for x in vms]
```

Comment:
I only wanted to avoid pylint W0110 message (http://pylint-messages.wikidot.com/all-messages). 
It is also advised here https://docs.quantifiedcode.com/python-anti-patterns/readability/using_map_or_filter_where_list_comprehension_is_possible.html
and here https://google.github.io/styleguide/pyguide.html?showone=Deprecated_Language_Features#Deprecated_Language_Features
Personally I find LC easier to read (but not always).